### DISCUSSION TITLE
Missing Achievements?

### DISCUSSION THREAD

**Discussion by Ben-Sicat** [community/community#55846] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My profile used to have a few badges now it diluted to 1 achivement what happend?
![image](https://github.com/community/community/assets/93244935/085e2e66-448d-466b-b3d8-078999c3449e)

**Comment by iamreiyn** [comment, marked as answer]
It's possible that you may have previously set your achievements to be hidden.
You can check the list of earned achievements here: https://github.com/Ben-Sicat?tab=achievements

If you find your hidden achievements listed there, simply click on the button (as shown below) to display them. Thank you.

![Achievements](https://i.imgur.com/8s9w5EQ.png)
